dogfood/main.tf

```
@@ -31,17 +31,18 @@ locals {
     "sa-saopaulo"   = "tcp://oberstein-sao-cdr-dev.tailscale.svc.cluster.local:2375"
   }
 
-  repo_dir       = replace(data.coder_parameter.repo_dir.value, "/^~\\//", "/home/coder/")
+  repo_base_dir  = replace(data.coder_parameter.repo_base_dir.value, "/^~\\//", "/home/coder/")
+  repo_dir       = module.git-clone.repo_dir
   container_name = "coder-${data.coder_workspace.me.owner}-${lower(data.coder_workspace.me.name)}"
   registry_name  = "codercom/oss-dogfood"
   jfrog_host     = replace(var.jfrog_url, "https://", "")
 }
 
-data "coder_parameter" "repo_dir" {
+data "coder_parameter" "repo_base_dir" {
   type        = "string"
-  name        = "Coder Repository Directory"
-  default     = "~/coder"
-  description = "The directory specified will be created and [coder/coder](https://github.com/coder/coder) will be automatically cloned into it 🪄."
+  name        = "Coder Repository Base Directory"
+  default     = "~"
+  description = "The directory specified will be created and [coder/coder](https://github.com/coder/coder) will be automatically cloned into `coder` subdirectory in it 🪄."
```

```suggestion
  description = "The directory specified will be created (if missing) and [coder/coder](https://github.com/coder/coder) will be automatically cloned into `[base directory]/coder` 🪄."
```
Alternate take, original is fine too.